frontend/src/helpers/sorting.ts

```diff
@@ -0,0 +1,28 @@
+import {Group, GroupUser, PunishmentType} from "./types";
+
+export function sortGroups(groups: Group[]) {
+    return groups.sort((a, b) => a.name_short.localeCompare(b.name_short));
+}
+
+export function sortGroupUsers(members: GroupUser[], punishmentTypes: PunishmentType[]) {
+    const punishmentTypeMap = punishmentTypes.reduce((acc, curr) => {
+        acc[curr.punishment_type_id] = curr;
+        return acc;
+    }, {} as { [id: number]: PunishmentType });
+    return members.sort((a, b) => {
+        const punishment_value = (user: GroupUser) => user.punishments.reduce(
```

Comment:
Pretty sure we talked about sorting this list based on how many punishments you currently have, not by the total value.

Reply:
true